### DISCUSSION TITLE
[A-L] (8 Febbraio 2022) Esercizio 2

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#272] [Esami A-L]
![image](https://user-images.githubusercontent.com/69389461/153032386-0effbe90-3fc8-44e4-a526-b47ec649d105.png)

**Comment by i0nut02** [comment]
1. F perché f non può essere iniettiva
2. V  
3. V

**Comment by matypist** [comment]
_Identico a #74_
